checkpoint/checkpoint_compare.sh

```diff
@@ -9,13 +9,14 @@ PROJECT_ROOT=$(dirname ${CHECKPOINT_ROOT})
 DOCKER_ACCOUNT=uscictdocker
 DOCKER_REPO=mentorpal-classifier
 DOCKER_TAG=latest
-DOCKER_IMAGE=${DOCKER_ACCOUNT}/${DOCKER_REPO}:${DOCKER_TAG}
 
-docker build -t ${DOCKER_IMAGE} .
+DOCKER_IMAGE=${DOCKER_ACCOUNT}/${DOCKER_REPO}:${DOCKER_TAG}
+DOCKER_CONTAINER=mentorpal-classifier
 
 docker run \
```

Comment:
Can we move `compare_checkpoints.py` and any scripts used for comparing checkpoints (but not at runtime) to `bin`?

The reasons are that we should be looking to keep everything not essential to production/runtime out of the base classifier image. Eventually we want to get rid of that root `src` folder somehow too, but for now an easy start is to move bin scripts elsewhere.